reportportal/values.yaml

```diff
@@ -155,13 +155,8 @@ serviceapi:
   jvmArgs: "-Djava.security.egd=file:/dev/./urandom -XX:+UseG1GC -XX:MinRAMPercentage=60.0 -XX:InitiatingHeapOccupancyPercent=70 -XX:MaxRAMPercentage=90.0 -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp"
 
   ## @param serviceapi.queues define the number for the queues
-  ## @param serviceapi.queues.totalNumber define the total number of queues
-  ## @param serviceapi.queues.perPodNumber define the number of queues per pod
-  ## How to calculate: @param serviceapi.queues.perPodNumber = @param serviceapi.queues.totalNumber / @ param serviceapi.replicaCount
```

Comment:
Hi,
Why has this been removed? Is it recommended for the total queue number to be equal to the per-pod number now?